CHANGELOG.md

```diff
@@ -5,6 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.1.2] - 2024-03-11
+
+* Be more permissive with user attributes on create/update
+* Camelize user params on update
+* Return user object on update (same as create)
+* Add activesupport to enable deep camelize of JSON objects
+* Let exceptions move up the call stack
```

Comment:
`Let exceptions move up the call stack` would be handy, indeed. However, this means you are going to break the API of this library. This should be considered as a breaking change in a major version.